### ISSUE TITLE
Getting error loadin ledger plugin after updating from 3.3.5 to 3.3.7

### ISSUE THREAD

**Issue by Transisto** [spesmilo/electrum#5492]
![image](https://user-images.githubusercontent.com/388716/60746646-e7193d80-9f4d-11e9-9826-d5923f607b3f.png)

Then get asked for opening the wallet file (ledger based) then electrum crashes.

![image](https://user-images.githubusercontent.com/388716/60746676-234c9e00-9f4e-11e9-82fd-f5a00cfab784.png)

**Comment by SomberNight** [comment]
Is this Windows?
Try opening another wallet file, [enable debug logging](https://github.com/spesmilo/electrum-docs/blob/master/faq.rst#how-to-enable-debug-logging) and then reproduce the crash and upload logs.

**Comment by Transisto** [comment]
`20190705T224024.722190Z |     INFO | logging | Electrum version: 3.3.5 - https://electrum.org - https://github.com/spesmilo/electrum
20190705T224024.763338Z |     INFO | logging | Python version: 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 bit (Intel)]. On platform: Windows-10-10.0.17763
20190705T224024.763338Z |     INFO | logging | Logging to file: C:\Users\edm\AppData\Roaming\Electrum\logs\electrum_log_20190705T224024Z_13736.log
20190705T224024.763338Z |     INFO | logging | Log filters: verbosity '', verbosity_shortcuts ''
20190705T224025.785789Z |     INFO | daemon | failed to connect to JSON-RPC server: [WinError 10061] No connection could be made because the target machine actively refused it
20190705T224025.789738Z |     INFO | plugin.Plugins | registering hardware coldcard: ('hardware', 'coldcard', 'Coldcard Wallet')
20190705T224025.791738Z |     INFO | plugin.Plugins | registering hardware digitalbitbox: ('hardware', 'digitalbitbox', 'Digital Bitbox wallet')
20190705T224025.795736Z |     INFO | plugin.Plugins | registering hardware keepkey: ('hardware', 'keepkey', 'KeepKey wallet')
20190705T224025.797736Z |     INFO | plugin.Plugins | registering hardware ledger: ('hardware', 'ledger', 'Ledger wallet')
20190705T224025.800735Z |     INFO | plugin.Plugins | registering hardware safe_t: ('hardware', 'safe_t', 'Safe-T mini wallet')
20190705T224025.801734Z |     INFO | plugin.Plugins | registering hardware trezor: ('hardware', 'trezor', 'Trezor wallet')
20190705T224025.802734Z |     INFO | plugin.Plugins | registering wallet type ('2fa', 'trustedcoin')
20190705T224025.803734Z |    DEBUG | util.profiler | Plugins.__init__ 0.0170
20190705T224025.807733Z |     INFO | network | blockchains [0, 570758]
20190705T224025.808732Z |     INFO | exchange_rate.FxThread | using exchange BitcoinAverage
20190705T224025.809732Z |     INFO | network | starting network
20190705T224025.809732Z |     INFO | network | connecting to dedi.jochen-hoenicke.de:50002:s as new interface
20190705T224025.810731Z |    DEBUG | util.profiler | Daemon.__init__ 0.0070
20190705T224025.812742Z |     INFO | exchange_rate.BitcoinAverage | getting fx quotes for CAD
20190705T224026.045655Z |     INFO | gui.qt.history_list | could not import electrum.plot. This feature needs matplotlib to be installed.
20190705T224026.244591Z |     INFO | interface.[hsmiths5mjk6uijs.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224026.244591Z |     INFO | interface.[3smoooajg7qqac2y.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224026.245591Z |     INFO | network | couldn't launch iface hsmiths5mjk6uijs.onion:50002:s -- CancelledError()
20190705T224026.245591Z |     INFO | network | couldn't launch iface 3smoooajg7qqac2y.onion:50002:s -- CancelledError()
20190705T224026.263586Z |     INFO | interface.[electrum2.eff.ro:50002] | disconnecting due to: gaierror(11001, 'getaddrinfo failed')
20190705T224026.263586Z |     INFO | network | couldn't launch iface electrum2.eff.ro:50002:s -- CancelledError()
20190705T224026.291606Z |     INFO | interface.[hsmiths4fyqlw5xw.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224026.291606Z |     INFO | network | couldn't launch iface hsmiths4fyqlw5xw.onion:50002:s -- CancelledError()
20190705T224026.300591Z |     INFO | interface.[qtornadoklbgdyww.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224026.300591Z |     INFO | network | couldn't launch iface qtornadoklbgdyww.onion:50002:s -- CancelledError()
20190705T224026.422552Z |     INFO | exchange_rate.BitcoinAverage | received fx quotes
20190705T224026.548495Z |     INFO | interface.[vmd27610.contaboserver.net:50002] | disconnecting due to: gaierror(11001, 'getaddrinfo failed')
20190705T224026.548495Z |     INFO | network | couldn't launch iface vmd27610.contaboserver.net:50002:s -- CancelledError()
20190705T224026.736434Z |     INFO | interface.[e3.keff.org:50002] | connection established. version: ['ElectrumX 1.10.0', '1.4']
20190705T224026.737434Z |     INFO | interface.[fortress.qtornado.com:50002] | connection established. version: ['ElectrumX 1.12', '1.4']
20190705T224026.852398Z |     INFO | interface.[e3.keff.org:50002] | set blockchain with height 584050
20190705T224026.852398Z |     INFO | interface.[e3.keff.org:50002] | skipping header 584050
20190705T224026.855397Z |     INFO | interface.[fortress.qtornado.com:50002] | set blockchain with height 584050
20190705T224026.856396Z |     INFO | interface.[fortress.qtornado.com:50002] | skipping header 584050
20190705T224027.197288Z |    DEBUG | util.profiler | ElectrumGui.__init__ 1.1486
20190705T224027.197288Z |     INFO | storage.WalletStorage | wallet path c:\users\edm\appdata\roaming\electrum\wallets\default_wallet
20190705T224027.198288Z |    DEBUG | util.profiler | JsonDB.load_transactions 0.0000
20190705T224027.200287Z |    DEBUG | util.profiler | AddressSynchronizer.load_local_history 0.0000
20190705T224027.200287Z |    DEBUG | util.profiler | AddressSynchronizer.check_history 0.0000
20190705T224027.569174Z |     INFO | interface.[yuio.top:50002] | disconnecting due to: ConnectionRefusedError(10061, "Connect call failed ('127.0.0.1', 50002)")
20190705T224027.572168Z |     INFO | network | couldn't launch iface yuio.top:50002:s -- CancelledError()
20190705T224027.798095Z |    DEBUG | util.profiler | Deterministic_Wallet.try_detecting_internal_addresses_corruption 0.0920
20190705T224027.798095Z |    DEBUG | util.profiler | ElectrumWindow.load_wallet 0.1499
20190705T224027.842081Z |     INFO | gui.qt.history_list.HistoryModel | refreshing... reason: update_tabs
20190705T224027.843082Z |    DEBUG | util.profiler | Abstract_Wallet.get_full_history 0.0000
20190705T224027.844102Z |    DEBUG | util.profiler | HistoryModel.refresh 0.0020
20190705T224027.869072Z |     INFO | interface.[vmd30612.contaboserver.net:50002] | disconnecting due to: gaierror(11001, 'getaddrinfo failed')
20190705T224027.869072Z |     INFO | network | couldn't launch iface vmd30612.contaboserver.net:50002:s -- CancelledError()
20190705T224027.913217Z |     INFO | gui.qt.update_checker.UpdateCheckThread | valid sig for version announcement '3.3.7' from address '13xjmVAB1EATPP8RshTE8S8sNwwSUM9p1P'
20190705T224035.390265Z |     INFO | storage.WalletStorage | wallet path c:\users\edm\appdata\roaming\electrum\wallets\yap3segwitcompatible3
20190705T224035.399261Z |    DEBUG | util.profiler | JsonDB.load_transactions 0.0010
20190705T224035.415255Z |    ERROR | gui.qt.ElectrumGui | 
Traceback (most recent call last):
  File "electrum\plugin.py", line 112, in load_plugin
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Program Files (x86)\Electrum\electrum\plugins\ledger\qt.py", line 12, in <module>
    from ..hw_wallet.qt import QtHandlerBase, QtPluginBase
  File "C:\Program Files (x86)\Electrum\electrum\plugins\hw_wallet\qt.py", line 34, in <module>
    from electrum.gui.qt.util import (read_QIcon, WWLabel, OkButton, WindowModalDialog,
ImportError: cannot import name 'char_width_in_lineedit'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "electrum\gui\qt\__init__.py", line 227, in start_new_window
  File "electrum\daemon.py", line 261, in load_wallet
  File "electrum\wallet.py", line 1861, in __new__
  File "electrum\wallet.py", line 1692, in __init__
  File "electrum\wallet.py", line 1549, in __init__
  File "electrum\wallet.py", line 215, in __init__
  File "electrum\address_synchronizer.py", line 83, in __init__
  File "electrum\wallet.py", line 239, in load_and_cleanup
  File "electrum\wallet.py", line 1700, in load_keystore
  File "electrum\keystore.py", line 735, in load_keystore
  File "electrum\keystore.py", line 718, in hardware_keystore
  File "electrum\plugin.py", line 189, in dynamic_constructor
  File "electrum\plugin.py", line 197, in get_plugin
  File "electrum\plugin.py", line 115, in load_plugin
Exception: Error loading ledger plugin: ImportError("cannot import name 'char_width_in_lineedit'",)
20190705T224047.247624Z |     INFO | interface.[elx01.knas.systems:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('192.168.100.100', 50002)")
20190705T224047.247624Z |     INFO | network | couldn't launch iface elx01.knas.systems:50002:s -- CancelledError()
20190705T224047.495985Z |     INFO | interface.[bauerjda5hnedjam.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224047.495985Z |     INFO | network | couldn't launch iface bauerjda5hnedjam.onion:50002:s -- CancelledError()
20190705T224047.502979Z |     INFO | interface.[helicarrier.bauerj.eu:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('5.135.68.240', 50002)")
20190705T224047.502979Z |     INFO | interface.[ulrichard.ch:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('151.248.221.217', 50002)")
20190705T224047.503966Z |     INFO | network | couldn't launch iface helicarrier.bauerj.eu:50002:s -- CancelledError()
20190705T224047.503966Z |     INFO | network | couldn't launch iface ulrichard.ch:50002:s -- CancelledError()
20190705T224047.554067Z |     INFO | interface.[icarus.tetradrachm.net:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('209.99.40.222', 50002)")
20190705T224047.554067Z |     INFO | network | couldn't launch iface icarus.tetradrachm.net:50002:s -- CancelledError()
20190705T224047.783312Z |     INFO | interface.[electrum-server.ninja:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('145.239.44.204', 50002)")
20190705T224047.783312Z |     INFO | network | couldn't launch iface electrum-server.ninja:50002:s -- CancelledError()
20190705T224048.147111Z |     INFO | interface.[btc.usebsv.com:50006] | connection established. version: ['ElectrumX 1.12', '1.4']
20190705T224048.160235Z |     INFO | interface.[currentlane.lovebitco.in:50002] | connection established. version: ['ElectrumX 1.12', '1.4']
20190705T224048.220745Z |     INFO | interface.[77.20.219.157:50001] | connection established. version: ['ElectrumX 1.11.0', '1.4']
20190705T224048.267063Z |     INFO | interface.[btc.usebsv.com:50006] | set blockchain with height 584050
20190705T224048.267063Z |     INFO | interface.[btc.usebsv.com:50006] | skipping header 584050
20190705T224048.280123Z |     INFO | interface.[currentlane.lovebitco.in:50002] | set blockchain with height 584050
20190705T224048.280123Z |     INFO | interface.[currentlane.lovebitco.in:50002] | skipping header 584050
20190705T224048.340336Z |     INFO | interface.[77.20.219.157:50001] | set blockchain with height 584050
20190705T224048.341336Z |     INFO | interface.[77.20.219.157:50001] | skipping header 584050
20190705T224049.109515Z |     INFO | interface.[rbx.curalle.ovh:50002] | disconnecting due to: TimeoutError(10060, "Connect call failed ('176.31.252.219', 50002)")
20190705T224049.109515Z |     INFO | network | couldn't launch iface rbx.curalle.ovh:50002:s -- CancelledError()
20190705T224049.411983Z |     INFO | interface.[dedi.jochen-hoenicke.de:50002] | disconnecting due to: SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)')
20190705T224049.411983Z |     INFO | network | couldn't launch iface dedi.jochen-hoenicke.de:50002:s -- CancelledError()
20190705T224049.554846Z |     INFO | network | connecting to dedi.jochen-hoenicke.de:50002:s as new interface
20190705T224049.685433Z |     INFO | interface.[bauerjhejlv6di7s.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224049.685433Z |     INFO | network | couldn't launch iface bauerjhejlv6di7s.onion:50002:s -- CancelledError()
20190705T224049.931569Z |     INFO | interface.[shogoth.no-ip.info:50002] | connection established. version: ['ElectrumX 1.12', '1.4']
20190705T224050.055075Z |     INFO | interface.[shogoth.no-ip.info:50002] | set blockchain with height 584050
20190705T224050.055075Z |     INFO | interface.[shogoth.no-ip.info:50002] | skipping header 584050
20190705T224056.126305Z |     INFO | network | couldn't launch iface electrum2.villocq.com:50002:s -- TimeoutError()
20190705T224056.385005Z |     INFO | interface.[bitcoin3nqy3db7c.onion:50002] | disconnecting due to: ErrorGettingSSLCertFromServer(gaierror(11001, 'getaddrinfo failed'),)
20190705T224056.385968Z |     INFO | network | couldn't launch iface bitcoin3nqy3db7c.onion:50002:s -- CancelledError()
20190705T224057.046931Z |     INFO | interface.[electrum3.hodlister.co:50002] | connection established. version: ['ElectrumX 1.10.0', '1.4']
20190705T224057.174453Z |     INFO | interface.[electrum3.hodlister.co:50002] | set blockchain with height 584050
20190705T224057.174453Z |     INFO | interface.[electrum3.hodlister.co:50002] | skipping header 584050`

**Comment by Transisto** [comment]
I've reinstaller electrum and it's working great. Sorry I have so many electrum installed in every mode, Portable standalone, I won't be able to take the time to pin-point the source of the problem.

TLDR, it's solved, Closing.

**Comment by SomberNight** [comment]
Well looking at the log it seems you have an installation of mixed 3.3.5 and 3.3.7 files

**Comment by Transisto** [comment]
yeah it was using a standalone 3.3.5 into the now 3.3.7 folder.
